cloud.foundry.cli/src/test/resources/basic/app.yml

```diff
@@ -0,0 +1,15 @@
+name: testApp
```

Comment:
Is this a file for testing the Creation of an app in an edn to end way?
If yes, I think we should create an extra folder with YAML files for our end to end tests. Otherwise we may get confused which files are needed for automated tests and which for an template on how to use the ap, etc.